pages/search.js

```diff
@@ -12,12 +12,12 @@ import Input from "@components/form/Input";
 
 export async function getServerSideProps() {
   let data = {
-    users: [],
+    initialRandomUsers: [],
     tags: [],
   };
   try {
-    const res = await fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/users`);
-    data.users = await res.json();
+    const res = await fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/users?`+new URLSearchParams({initialRandomUsers:true}));
```

Comment:
Mixed styles here...

```suggestion
    const res = await fetch(`${process.env.NEXT_PUBLIC_BASE_URL}/api/users?${new URLSearchParams({initialRandomUsers:true})}`);
```